### DISCUSSION TITLE
能否支持v2rayA

### DISCUSSION THREAD

**Discussion by sillypy** [vvbbnn00/WARP-Clash-API#89] [Ideas]
家里软路由装的v2rayA，这个项目能否支持一下

**Comment by vvbbnn00** [comment]
你好，目前v2rayA并不支持wireguard协议，不过，我已在新版本中增加了针对v2ray订阅格式的支持。
若未来v2ray支持wireguard协议，可以通过订阅链接直接使用。

**Reply by WyInnovate** [reply]
好像v2rayN要通过自定义配置来支持

**Comment by zhangyupy** [comment]
新版的v2ray已经支持wirguard协议了

**Comment by vvbbnn00** [comment]
> 新版的v2ray已经支持wirguard协议了

你好，请问是哪一个版本的v2ray，是指xray内核吗，还是某个GUI客户端，是否方便提供项目地址以便适配？谢谢

**Comment by sillypy** [comment]
> > 新版的v2ray已经支持wirguard协议了
> 
> 你好，请问是哪一个版本的v2ray，是指xray内核吗，还是某个GUI客户端，是否方便提供项目地址以便适配？谢谢

就是v2rayA(https://v2raya.org/), linux下的，我是家里电视盒子当软路由用

**Comment by vvbbnn00** [comment]
> > > 新版的v2ray已经支持wirguard协议了
> > 
> > 
> > 你好，请问是哪一个版本的v2ray，是指xray内核吗，还是某个GUI客户端，是否方便提供项目地址以便适配？谢谢
> 
> 就是v2rayA(https://v2raya.org/), linux下的，我是家里电视盒子当软路由用

这个我下载了上周发布的windows版本做了测试，似乎还不支持wireguard

**Comment by sillypy** [comment]
> > > > 新版的v2ray已经支持wirguard协议了
> > > 
> > > 
> > > 你好，请问是哪一个版本的v2ray，是指xray内核吗，还是某个GUI客户端，是否方便提供项目地址以便适配？谢谢
> > 
> > 
> > 就是v2rayA(https://v2raya.org/), linux下的，我是家里电视盒子当软路由用
> 
> 这个我下载了上周发布的windows版本做了测试，似乎还不支持wireguard

期待支持v2ray订阅，毕竟clash跑路了
